deny-not-main-or-master-revision/deny.rego

```diff
@@ -0,0 +1,11 @@
+package env0
+
+# METADATA
+# title: deny for non-main revisions
+# description: deny if deployment revision is not "main" or "master"
+deny[format(rego.metadata.rule())] {
+    allowed_revisions = { "main", "master" }
+	not allowed_revisions[input.deploymentRequest.revision]
```

Comment:
I just hate rego what kind of evil syntax is this

Reply:
that's how we test if a value is part of the set.
the not is for negation